File: JavaScript/1769. Minimum Number of Operations to Move All Balls to Each Box.js
/**
 * @param {string} boxes
 * @return {number[]}
 */
var minOperations = function(boxes) {
        var n = boxes.length;
        var res = new Array(n);
        for(var i = 0; i<n; i++){
            var temp = 0;
            for(var j = 0; j<n; j++){
                if(boxes[j]=='1'){
                    temp+=Math.abs(i-j);
                }
            }
            res[i] = temp;
        }
        return res;
};

File: C#/1769. Minimum Number of Operations to Move All Balls to Each Box.cs
public class Solution {
    public int[] MinOperations(string boxes) {
        int n = boxes.Length;
        int[] res = new int[n];
        for(int i = 0; i<n; i++){
            int temp = 0;
            for(int j = 0; j<n; j++){
                if(boxes[j]=='1'){
                    temp+=Math.Abs(i-j);
                }
            }
            res[i] = temp;
        }
        return res;
    }
}
